nixos/modules/security/apparmor.nix

```diff
@@ -29,6 +39,577 @@ in
    config = mkIf cfg.enable {
      environment.systemPackages = [ pkgs.apparmor-utils ];
 
+     # Needed globals and abstractions that can be included in profiles
+     environment.etc."/apparmor.d/abstractions".source = pkgs.apparmor-profiles + "/etc/apparmor.d/abstractions";
+     environment.etc."/apparmor.d/tunables".source = pkgs.apparmor-profiles + "/etc/apparmor.d/tunables";
+
+     environment.etc."apparmor/parser.conf".text = ''
+      Optimize=compress-fast
+      ${cfg.extraParserConfig}
+    '';
+
+     # Pretty much everything in here, is there to
+     # prevent apparmor utilities from crashing.
+     environment.etc."apparmor/logprof.conf".text =
```

Comment:
Where does these configuration comes from? Could we directly take it from the package instead of copying it to here?

Reply:
Well the apparmor and apparmor-profiles package does not provide these files. I have them from here https://gitlab.com/apparmor/apparmor/-/tree/master/utils%2Ftest should we create a package only for these files? Seems a bit pointless to me.